### PR TITLE
Update to latest dependency versions

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to d758edb433773e0c812b61b7c4214f3ff294842a
js: 17.71kb decrease (-4.62kb/gzip)
css: 4.66kb decrease (-0.49kb/gzip)

File: src/scss/_color-use-cases.scss [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-
-@include oColorsSetUseCase('o-cookie-message-alternative', 'border', 'jade');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'text', 'white');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'background', 'jade');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'hover', 'jade-black-70');
-@include oColorsSetUseCase('o-cookie-message-alternative-link', 'text', 'black');
-@include oColorsSetUseCase('o-cookie-message-alternative-link', 'background', 'white');
+@include oColorsSetColor('o-cookie-message/jade-black-70', oColorsMix('jade', 'black', 70));
+
+@include oColorsSetUseCase('o-cookie-message/alternative', (
```
Comment (notlee) [inline src/scss/_color-use-cases.scss]: This is correct but we don't need to share these colours with any other component so it'd be better to make these private Sass variables

File: src/scss/_mixins.scss [hunk 2]
```diff
@@ -107,10 +107,10 @@
 	}
 
 	.o-cookie-message__button {
-		@include oButtons(
-			$size: 'big',
-			$theme: 'primary'
-		);
+		@include oButtonsContent((
+			'size': 'big',
+			'theme': 'primary'
```
Comment (notlee) [inline src/scss/_mixins.scss]: 'primary' and 'secondary' are now button "types". Themes are "inverse", "mono", "b2c", which can be used to modify the look of the two button types

File: src/scss/_mixins.scss [hunk 3]
```diff
@@ -120,8 +120,7 @@
 	.o-cookie-message {
 		.o-cookie-message__heading {
 			h1 {
-				@include oTypographySans($scale: 3);
-				@include oTypographyBold($font: 'sans');
+				@include oTypographySans($scale: 3, $weight: 'bold');
```
Comment (notlee) [inline src/scss/_mixins.scss]:
```suggestion
				@include oTypographySans($scale: 3, $weight: 'semibold');
```

File: src/scss/_mixins.scss [hunk 4]
```diff
@@ -190,31 +187,37 @@
...
-				background: oColorsGetUseCase(o-cookie-message-alternative-button, text),
-				hover: oColorsGetUseCase(o-cookie-message-alternative-button, hover),
-				colorizer: primary
-			));
+			@include oButtonsContent((
+				theme: (
+					accent: oColorsGetUseCase(o-cookie-message/alternative-button, background),
+					background: oColorsGetUseCase(o-cookie-message/alternative-button, text),
+					hover: oColorsGetUseCase(o-cookie-message/alternative-button-hover, background),
+					colorizer: primary
```
Comment (notlee) [inline src/scss/_mixins.scss]: the button type is given with `type` key in the root of this options map, `accent` is `color` and `background` is `context`

**Comment by origamiserviceuser** [comment]
o-cookie-message bundle size difference from 4.7.3 to 7341eb71c0d89fe6c1fb3b6d63a20ac02da36598
js: 17.71kb decrease (-4.63kb/gzip)
css: 5.1kb decrease (-0.44kb/gzip)

File: src/scss/_color-use-cases.scss (hunk 1 above)
Comment (chee) [reply src/scss/_color-use-cases.scss]: oh that's good news, i didn't understand why these were in a usecase when it seemed less complex (and easier to read!) to use a variable. very glad to do that

**Pull request merged**